runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -668,7 +672,7 @@ static TR::Node *getStoreValueBaseNode(TR::Node *storeValueNode, TR::SymbolRefer
             }
          }
    else if (storeValueNode->getOpCode().isCall()
-            && storeValueNode->getSymbolReference() == loadFlattenableElementSymRef)
+            && (symRefTab->isNonHelper(storeValueNode->getSymbolReference(), loadFlattenableArrayElementNonHelper) || (storeValueNode->getSymbolReference() == loadFlattenableElementHelperSymRef)))
```

Comment:
Changing this line to the following would allow you to drop the `loadFlattenableArrayElementNonHelper` and `loadFlattenableElementHelperSymRef` arguments to this function.

```
             && (symRefTab->isNonHelper(storeValueNode->getSymbolReference(), TR::SymbolReferenceTable::loadFlattenableArrayElementNonHelperSymbol) || (storeValueNode->getSymbolReference()->getReferenceNumber() == TR_ldFlattenableArrayElement)))
```

Reply:
Addressed in 446e692